packages/app/package.json

```
@@ -6,17 +6,30 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "msw": "npx msw init public/ --save",
+    "storybook": "storybook dev -p 6006",
```

```suggestion
    "storybook": "storybook dev -p 6007",
```
design system의 스토리북 port를 6006으로 사용하고 있으므로 다른 포트로 변경해주는건 어떨까요?